manager/integration/tests/test_metric.py

```diff
@@ -54,9 +58,9 @@
 def get_metrics(core_api, metric_node_id): # NOQA
     pods = core_api.list_namespaced_pod(namespace=LONGHORN_NAMESPACE,
                                         label_selector="app=longhorn-manager")
-    for pod in pods.items:
-        if pod.spec.node_name == metric_node_id:
-            manager_ip = pod.status.pod_ip
+    for po in pods.items:
```

Comment:
Is there a specific reason for renaming this?

Reply:
because I import the `pod()` function from common
```
from common import pod
```
so we can't use the same name here in the loop

Reply:
I feel like the function name should be a verb to avoid this kind of conflict.